packages/components/src/Stack/Stack.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { useStyles, system } from '@marigold/system';
+import { Box } from '../Box';
+import flattenChildren from 'react-flatten-children';
+
+type StackProps = {
+  space?: 0 | 4 | 8 | 16 | 24 | 32 | 40 | 48 | 56 | 64 | 88;
+  align?: 'left' | 'right' | 'center';
+};
+
+export const Stack = system<StackProps, 'div'>(
+  ({ space = 0, align = 'left', className, children, ...props }) => {
+    const classNames = useStyles({}, className);
```

Comment:
Does this do anything?

Reply:
No, but if I give custom classNames

Reply:
You mean the base css props? The box will have them anyway.

Reply:
No I mean custom user styling when i use <Stack classNames="my-custom-style">

Reply:
When you use? :D

Reply:
e.g. like this: `<Stack space={24} className={useStyles({ bg: '#f3f3f3' })}>`

Reply:
But in this case you get the className anyway, so you can just put the className on the inner div.

Reply:
ooooh so right